set.mm

```diff
@@ -459220,8 +459220,21 @@ SUPPLEMENTARY MATERIAL (USER'S MATHBOXES)
        5.  Mathboxes must be independent from one another (checked by "verify
        markup *").  If you need a theorem from another mathbox, typically it is
        moved to the main part of set.mm.  New users should consult with more
-       experienced users before doing this.  (Contributed by NM, 20-Feb-2007.)
-       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+       experienced users before doing this.
+
+       6.  If a contributor is no longer active, we will continue the usual
+       maintenance edits.  As time goes on, often theorems will be moved to
```

Comment:
Are removals of expired OLD proofs part of the usual maintenance edits? Or should we treat them differently if in a mathbox?

Reply:
That would be like minimization - the same in or out of a mathbox.

Reply:
I agree, I'll make a PR for their removal soon (unless someone else will do it before me).

Reply:
@tirix There are some OLD proofs in your mathbox missing an 'Obsolete as of ...' description, what do we do with those?

My proposal with OLD proofs would be to avoid exceptions: if they can be proven to be older than one year with the github history then even the ones missing a date should be removed as well. I think we should aim to be consistent with the meaning of OLD as "to be removed" (as the conventions page indicates), so if an author adds the OLD suffix to a label then he is sending the message that it can be deleted. If the intention is to keep an alternative version of a theorem, the ALT suffix is available for that purpose.

Reply:
If they are missing an 'Obsolete as of' description, perhaps they should be renamed to something other than xxxOLD instead of removed. But I didn't look through them, so I don't have a theorem-by-theorem opinion about what to do about each one.

Reply:
> @tirix There are some OLD proofs in your mathbox missing an 'Obsolete as of ...' description, what do we do with those?

Yes, I agree there should not be exceptions. It looks like I have taken the habit to append "OLD" to theorem for which I had a better version. But in many cases the formulas are different. Let's review those:
- `ogrpinvOLD` : this is superseded by `ogrpinv0le`, which is more powerful (equivalence, instead of just one direction). So it can be deleted.
- `xrge0tmdOLD`: proof has been shortened, but the proof method has changed. Maybe the old proof should be kept as 'ALT'.
- `isrnsigaOLD` : proof has been shortened, can be deleted.
- `measdivcstOLD` : this is a different formulation as `measdivcst`, not a drop-in replacement, and it is actually still used in `probmeasb`, so first the proof of `probmeasb` should be adapted to the new notation, then this version can be deleted. Or I just keep it with a different name.
- `probfinmeasbOLD` : same as `measdivcstOLD`: this is a different formulation as `measdivcst`, not a drop-in replacement.
- `axtglowdim2OLD` and `axtgupdim2OLD` : They are using the deprecated `TarskiG2D` notation which I chose to keep in my Mathbox. They have been superseded by the corresponding theorems with the `TarskiGDim>=` notation. I guess I was unsure of the future of the `TarskiGDim>=` notation. Maybe these should be renamed something different than with the `OLD` suffix.

I should do these housekeeping changes in my Mathbox, I'm not sure when I can come to it.
Are those inconvenient because of a tool or a plan to enforce specifics for the `OLD` theorems?

Reply:
> I should do these housekeeping changes in my Mathbox, I'm not sure when I can come to it.

Thank you for your exaustive answer. If it's ok for you, I can do these maintenance edits following your instructions.

> Are those inconvenient because of a tool or a plan to enforce specifics for the OLD theorems?

I'm very much in favor of the idea of developing a tool that, given a database, can check all OLD statement dates and automatically generate a copy of the database without those OLD theorems. While I'm not planning on working on it anytime soon, I intend to enforce the updated conventions from this PR, so that there won't be any obstacles in case this tool will be implemented. Besides that, I simply believe that keeping the database clean helps to avoid messiness and prevents unnecessary memory usage.


I'm a bit busy these days, so I'll make the housekeeping PR next week or two.

Reply:
> If it's ok for you, I can do these maintenance edits

Sure, thanks! And I would get to review them anyway.